RecoTracker/DeDx/test/BuildFile.xml

```diff
@@ -0,0 +1,26 @@
+<use name="TrackingTools/Records"/>
+<use name="RecoTracker/Record"/>
+<use name="FWCore/Utilities"/>
+<use name="DataFormats/TrackReco"/>
+<use name="RecoTracker/TrackProducer"/>
+<use name="TrackingTools/TransientTrack"/>
+<use name="clhep"/>
+<use name="FWCore/Framework"/>
+<use name="FWCore/ParameterSet"/>
+<use name="Geometry/TrackerGeometryBuilder"/>
+<use name="Geometry/Records"/>
+<use name="MagneticField/Records"/>
+<use name="MagneticField/Engine"/>
+<use name="SimDataFormats/Vertex"/>
+<use name="TrackingTools/PatternTools"/>
+<use name="root"/>
+
+<library file="ProbQXYAna.cc" name="ProbQXYAna">
+  <flags EDM_PLUGIN="1"/>
+</library>
+
+<environment>
+  <bin file="testDeDxScripts.cpp">
+    <flags TEST_RUNNER_ARGS="/bin/bash RecoTracker/TrackProducer/test testProbQXY.sh"/>
```

Comment:
```suggestion
    <flags TEST_RUNNER_ARGS="/bin/bash RecoTracker/DeDx/test testProbQXY.sh"/>
```

the error message from the [log](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-ecf9e7/20847/unitTests/src/RecoTracker/DeDx/test/testDeDxScripts/testing.log) is:
```
 
===== Test "testDeDxScripts" ====
argument 0: testDeDxScripts
argument 1: /bin/bash
argument 2: RecoTracker/TrackProducer/test
argument 3: testProbQXY.sh
shell is: /bin/bash
Current directory is: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100
topdir is: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100
testdir is: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100/src/RecoTracker/TrackProducer/test
tmpdir is: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100/tmp/slc7_amd64_gcc900
testbin is: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100/test/slc7_amd64_gcc900
Running script: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100/src/RecoTracker/TrackProducer/test/testProbQXY.sh
sh: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_2_X_2021-11-29-1100/src/RecoTracker/TrackProducer/test/testProbQXY.sh: No such file or directory
status = 32512

---> test testDeDxScripts had ERRORS
TestTime:2
^^^^ End Test testDeDxScripts ^^^^
```